sdk/program/src/serialize_utils/cursor.rs

```diff
@@ -50,6 +56,27 @@ pub(crate) fn read_i64<T: AsRef<[u8]>>(cursor: &mut Cursor<T>) -> Result<i64, In
     Ok(i64::from_le_bytes(buf))
 }
 
+pub(crate) fn read_pubkey_into(
+    cursor: &mut Cursor<&[u8]>,
+    pubkey: *mut Pubkey,
+) -> Result<(), InstructionError> {
+    match cursor.fill_buf() {
+        Ok(buf) if buf.len() >= PUBKEY_BYTES => {
+            // Safety: `buf` is guaranteed to be at least `PUBKEY_SIZE` bytes
```

Comment:
nit: ... to be at least `PUBKEY_BYTES` bytes ...

Reply:
done